ballerina/client.bal

```diff
@@ -1,3 +1,6 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
```

Comment:
We need the license header here.

Let's create the license.txt using the below content and use the `--license` flag in the openapi tool when generating the client

https://github.com/ballerina-platform/module-ballerinax-hubspot.crm.object.tickets/blob/main/docs/license.txt